src/main/app/src/app/zaken/zaken-mijn/zaken-mijn.component.html

```diff
@@ -75,7 +75,7 @@
                     <div fxLayout="row wrap" fxLayoutGap="10px grid" fxFlexFill>
                         <ng-container *ngFor="let column of dataSource.columns | visibleColumn:false">
                             <zac-static-text fxFlex="33" [label]="column.label | translate"
-                                             [value]="row[column.model] | translate | empty"></zac-static-text>
+                                             [value]="row[column.model] | translate"></zac-static-text>
```

Comment:
moet die [value] nog door de translate pipe? Kan best goed zijn hoor. Ik weet het niet.

Reply:
Ja dat was nodig omdat dit in een ngFor staat, er kunnen waardes tussen zitten die vertaald moeten worden. Dit gaat opgelost worden wanneer de werklijsten uitgeschreven worden, dan kunnen de specifieke velden vertaald worden.